doc/1/controllers/auth/login/index.md

```diff
@@ -26,7 +26,7 @@ public async Task<JObject> LoginAsync(
 |---------------|--------------------|--------------------------------------|
 | `strategy`    | <pre>string</pre>  | Strategy to use                      |
 | `credentials` | <pre>JObject</pre> | JObject representing the credentials |
-| `expiresIn`   | <pre>Int64</pre>   | Expiration time, in milliseconds     |
+| `expiresIn`   | <pre>TimeSpan?</pre>   | Token duration  |
```

Comment:
Maybe we can add a note on the behaviour when `expiresIn` is not set (default TTL from [kuzzlerc](https://github.com/kuzzleio/kuzzle/blob/master/.kuzzlerc.sample#L146) ) ?

Reply:
It's already there, in the `expiresIn` detailed section below this table: https://github.com/kuzzleio/sdk-csharp/blob/95e594b65ffaf655fdf16fd386e29deef309e01f/doc/1/controllers/auth/login/index.md#expiresin

Reply:
Oh ok, all good so :+1: